controllers/registerController.js

```diff
@@ -15,20 +15,26 @@ exports.postRegistraionInfo = (req, res, next) => {
     // create validation manually in server side, instead using required function in front end
     if (
         !name || 
-        name.trim() === '' ||
+        name.trim() === ''
+    ) {
+        res.status(422).json({ message: 'Please enter your name.'})
```

Comment:
Typically, it's best practice to use a short error message like `invalid-name` or `no-name`. So the front end (client) can check the error message easily (not very different, though!)